src/store/selectors/exchange.ts

```diff
@@ -384,4 +398,160 @@ export const getHaltedState = createSelector(state, allState => {
   return allState.halted
 })
 
+export interface UserVaults extends VaultEntry {
+  borrowed: string
+  collateral: string
+  deposited: number
+  depositedSign: string
+  cRatio: string
+  minCRatio: number
+  currentDebt: number
+  currentDebtSign: string
+  maxBorrow: string
+  interestRate: string
+  liquidationPrice: string
+}
+
+export const getUserVaults = createSelector(
```

Comment:
Overall this selector is massive. Does this cause performance problems ?

Reply:
I did not notice any major delays